static/app/views/projectInstall/overview.tsx

```diff
@@ -56,12 +59,18 @@ class ProjectInstallOverview extends AsyncComponent<Props, State> {
   };
 
   render() {
-    const {organization} = this.props;
+    const {organization, projects} = this.props;
     const {projectId} = this.props.params;
     const {keyList, showDsn} = this.state;
 
     const issueStreamLink = `/organizations/${organization.slug}/issues/#welcome`;
 
+    const project = projects.find(p => p.slug === projectId);
```

Comment:
Does `withProjects` guarantee that all projects will be returned? Even if org has hundreds of projects?

Reply:
I didn't want to convert this component just now to be able to use `useProjects` but this is done in a follow-up :) https://github.com/getsentry/sentry/pull/49303